DuckDuckGo/AppWidthObserver.swift

```diff
@@ -29,16 +29,20 @@ public class AppWidthObserver {
     }
     
     public static let shared = AppWidthObserver()
-    
-    private var currentWidth: CGFloat = 0
+
+    // mutable for testing
+    var isPad = UIDevice.current.userInterfaceIdiom == .pad
+    var currentWidth: CGFloat = 0
     
     public var isLargeWidth: Bool {
-        return currentWidth >= Constants.minPadWidth
+        return isPad && currentWidth >= Constants.minPadWidth
```

Comment:
Do you know if this can ever return false for iPads outside of the test suite after this change? AFAIK the [smallest dimension on any iPad](https://developer.apple.com/library/archive/documentation/DeviceInformation/Reference/iOSDeviceCompatibility/Displays/Displays.html) is 768 (iPad mini), and the `minPadWidth` here is `678`.

Does it make sense to drop the width check entirely? That said I could see an argument to be made for keeping it just as a safeguard since there's no guarantee that 768 will always be the smallest dimension on iPad, I'm happy either way here.

Reply:
The reason we size check is for the split screen scenarios.  The "thin" mode (ie < 678) needs to show the iPhone UI and we built it to basically mimic Safari's behaviour.